docroot/modules/custom/foia_webform/src/AgencyLookupService.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Drupal\foia_webform;
+
+use Drupal\node\Entity\Node;
+
+/**
+ * Class AgencyLookupService.
+ *
+ * @package Drupal\foia_webform
+ */
+class AgencyLookupService implements AgencyLookupServiceInterface {
+
+  /**
+   * Queries for an associated Agency Component given a form ID.
+   *
+   * @param string $webformId
+   *   The form ID.
+   *
+   * @return \Drupal\node\NodeInterface|null
+   *   The Agency Component object or NULL.
+   */
```

Comment:
Since this class implements an interface, we can just use the `inheritdoc` notation for the doc block for any methods that have doc blocks defined on the interface:
```
  /**
   * {inheritdoc}
   */
```